docs/src/man/split_apply_combine.md

```diff
@@ -35,25 +48,16 @@ julia> tail(iris)
 │ 5   │ 6.2         │ 3.4        │ 5.4         │ 2.3        │ virginica     │
 │ 6   │ 5.9         │ 3.0        │ 5.1         │ 1.8        │ virginica     │
 
-julia> by(iris, :Species, size)
+julia> by(iris, :Species, :PetalLength => mean)
 3×2 DataFrame
-│ Row │ Species       │ x1      │
-│     │ Categorical…⍰ │ Tuple…  │
-├─────┼───────────────┼─────────┤
-│ 1   │ setosa        │ (50, 5) │
-│ 2   │ versicolor    │ (50, 5) │
-│ 3   │ virginica     │ (50, 5) │
-
-julia> by(iris, :Species, df -> mean(df.PetalLength))
-3×2 DataFrame
-│ Row │ Species       │ x1      │
-│     │ Categorical…⍰ │ Float64 │
-├─────┼───────────────┼─────────┤
-│ 1   │ setosa        │ 1.462   │
-│ 2   │ versicolor    │ 4.26    │
-│ 3   │ virginica     │ 5.552   │
-
-julia> by(iris, :Species, df -> DataFrame(N = size(df, 1)))
+│ Row │ Species    │ PetalLength_mean │
+│     │ String⍰    │ Float64          │
+├─────┼────────────┼──────────────────┤
+│ 1   │ setosa     │ 1.462            │
+│ 2   │ versicolor │ 4.26             │
+│ 3   │ virginica  │ 5.552            │
+
+julia> by(iris, :Species, N = :Species => length) # Chosen column is arbitrary
```

Comment:
This form (i.e. keyword argument) is not covered by the rules listed above as this is not the same as passing a named tuple.